modules/web/src/app/node-data/template.html

```diff
@@ -281,7 +281,7 @@
 
       <km-number-stepper label="Min Replicas"
                          mode="all"
-                         hint="The minimum number of replicas for autoscaling. Minimum value can be 1"
+                         hint="The minimum number of replicas for autoscaling. Minimum value can be 0"
```

Comment:
I think on the backend, minimum value is `1` for this annotation. Can you please double check if that's the case?

Reply:
so the docs says must be greater than 0, so i will make it 1, https://docs.kubermatic.com/kubermatic/main/tutorials-howtos/kkp-autoscaler/cluster-autoscaler/#:~:text=(must%20be%20greater%20than%20zero)